htdocs/templates2/ocstyle/sys_main.tpl

```diff
@@ -332,6 +332,40 @@
             </div> <!-- page-container-1 -->
         </div> <!-- overall -->
 {/if}{*popup*}
+    {literal}
+        <script type="text/javascript">
+            // Set to the same value as the web property used on the site
+            var gaProperty = 'UA-59334952-4';
```

Comment:
Die Property ist eh "öffentlich" einsehbar. Wollte das in der neuen Theme-Struktur anpassen :)